gspread/__init__.py

```diff
@@ -28,14 +28,15 @@
 from .worksheet import Worksheet
 
 
-def authorize(credentials, client_class=Client):
+def authorize(credentials, client_factory=Client):
     """Login to Google API using OAuth2 credentials.
     This is a shortcut function which
-    instantiates `client_class`.
-    By default :class:`gspread.Client` is used.
+    instantiates using `client_factory`.
+    By default :class:`gspread.Client` is used (but could also use
+    :class:`gspread.BackoffClient` to avoid rate limiting).
 
-    :returns: `client_class` instance.
+    :returns: `client_factory` instance.
```

Comment:
This should be in `Client` and not `client_factory` and in fact the previous comment was completely wrong. 
The full sphinix docstring is `` :class:`gspread.client.Client` ``

Reply:
- I think `authorize()` is just top-level syntactic sugar over the functionality of `Client`. It needs to remain in the top-level. I can move it to `auth.py` if you want.

- Yes, I fixed the docstring to have the `:rtype:` and to clarify the explanatory string in the `:returns:`.

Reply:
I moved `authorize()` to `auth.py`